packages/tempus/src/Tempus_TimeStepControl_decl.hpp

```diff
@@ -32,36 +33,58 @@ namespace Tempus {
  *   - Maximum and minimum time index
  *   - Maximum and minimum time step size
  *   - Maximum and minimum error
- *   - Maximum and minimum order
  *   - Startup considerations (e.g., ramping)
  *   - Solution and/or diagnostic output
  *  Additional step control can be added through the step control observer,
  *  or inheriting from this class.
  *   - Stability limits (e.g., CFL number)
  *
- * Using TimeStepControlStrategy allows applications to define
- * their very own  strategy used to determine the next time step size (`getNextTimeStep()`). 
- * Applications can define multiple strategies and add it to a vector of strategies TimeStepControlStrategyComposite  using setTimeStepControlStrategy().
- * TimeStepControlStrategyComposite iterates over the list of strategies to determine
- * the "optimal" next time step size.
+ *  Using TimeStepControlStrategy allows applications to define their
+ *  very own strategy used to determine the next time step size
+ *  (`getNextTimeStep()`).  Applications can define multiple strategies
+ *  and add it to a vector of strategies TimeStepControlStrategyComposite
+ *  using setTimeStepControlStrategy().  TimeStepControlStrategyComposite
+ *  iterates over the list of strategies to determine the "optimal"
+ *  next time step size.
  *
  */
 template<class Scalar>
 class TimeStepControl
   : virtual public Teuchos::Describable,
-    virtual public Teuchos::ParameterListAcceptor,
     virtual public Teuchos::VerboseObject<Tempus::TimeStepControl<Scalar> >
 {
 public:
 
+  /// Default Constructor
+  TimeStepControl();
+
   /// Constructor
-  TimeStepControl(Teuchos::RCP<Teuchos::ParameterList> pList = Teuchos::null);
+  TimeStepControl(
+    Scalar              initTime,
+    Scalar              finalTime,
+    Scalar              minTimeStep,
+    Scalar              initTimeStep,
+    Scalar              maxTimeStep,
+    int                 initIndex,
+    int                 finalIndex,
+    Scalar              maxAbsError,
+    Scalar              maxRelError,
+    std::string         stepType,
```

Comment:
just a tiny suggestion: `stepType` should be first argument. I would want to know if I'm taking Constant/Variable before the `maxAbsError`

Reply:
Got it.  Moving it.